scripts.py

```diff
@@ -358,6 +370,12 @@ def run(self, uploader=None, cmd_args=None):
             annotator=ContentServerAnnotator()
         )
 
+        if parsed.search_url and parsed.search_index:
+            OPDSFeed.add_link_to_feed(feed=feed.feed,
+                                      rel="search",
+                                      href=feed_id + "/search",
```

Comment:
This requires the search endpoint to be "/search" on the domain that was passed in. I think that's safe for our use at least.